web-client/public/js/graph.js

```diff
@@ -1467,10 +1467,10 @@ export var drawGraph = function (network, sliderController, nodeColoring) {
     }
 
     // Configures sliderController
-    sliderController.addForce(simulation);
-    sliderController.configureForceHandlers();
+    grnState.simulation = simulation;
     sliderController.initializeDefaultForces();
 
+    // figure out this code???
```

Comment:
Additionally, I need a little bit of help figuring out what lines 1475 & 1476 are actually doing, in order to move that last function in sliders.js

Reply:
`value` is not being compared to anything; it is the target of an assignment statement. The value assigned to `value` is either `linkDistValidator($(item).val())` if the value of `slider` is `"link"`. Otherwise, `value` gets `chargeValidator($(item).val())`.

Let me know if that clears that up.